src/traces/table/attributes.js

```diff
@@ -0,0 +1,278 @@
+*/
+
+'use strict';
+
+var annAttrs = require('../../components/annotations/attributes');
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+
+    domain: {
+        x: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'number', min: 0, max: 1},
+                {valType: 'number', min: 0, max: 1}
+            ],
+            dflt: [0, 1],
+            description: [
+                'Sets the horizontal domain of this `table` trace',
+                '(in plot fraction).'
+            ].join(' ')
+        },
+        y: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'number', min: 0, max: 1},
+                {valType: 'number', min: 0, max: 1}
+            ],
+            dflt: [0, 1],
+            description: [
+                'Sets the vertical domain of this `table` trace',
+                '(in plot fraction).'
+            ].join(' ')
+        }
+    },
+
+    columnwidth: {
+        valType: 'number',
+        arrayOk: true,
+        dflt: null,
+        role: 'style',
+        description: 'The width of cells.'
+    },
+
+    columnorder: {
+        valType: 'data_array',
+        role: 'info',
+        description: [
+            'Specifies the rendered order of the data columns; for example, a value `2` at position `0`',
+            'means that column index `0` in the data will be rendered as the',
+            'third column, as columns have an index base of zero.'
+        ].join(' ')
+    },
+
+    header: {
+
+        values: {
+            valType: 'data_array',
+            role: 'info',
+            dflt: [],
+            description: [
+                'Dimension values. `values[n]` represents the value of the `n`th point in the dataset,',
+                'therefore the `values` vector for all dimensions must be the same (longer vectors',
+                'will be truncated). Each value must be a finite number.'
+            ].join(' ')
+        },
+
+        format: {
+            valType: 'data_array',
+            role: 'info',
+            dflt: [],
+            description: [
+                'Sets the cell value formatting rule using d3 formatting mini-language',
+                'which is similar to those of Python. See',
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+
+        prefix: {
+            valType: 'string',
+            arrayOk: true,
+            dflt: null,
+            role: 'style',
+            description: 'Prefix for cell values.'
+        },
+
+        suffix: {
+            valType: 'string',
+            arrayOk: true,
+            dflt: null,
+            role: 'style',
+            description: 'Suffix for cell values.'
+        },
+
+        height: {
+            valType: 'number',
+            dflt: 28,
```

Comment:
Interesting number. How did you come up with that?

Reply:
Just took a look at what'd look balanced with a font height of 12px although it could as well be computed
![image](https://user-images.githubusercontent.com/1548516/31135780-54918dc6-a866-11e7-8a51-2a5af96a164e.png)